setup.py

```diff
@@ -27,7 +27,7 @@
         'papyrus',
         'GeoAlchemy2',
         'OWSLib',
-        'fpdf',
+        'pypdftk',
```

Comment:
Is it used ???

I saw that you rather use a subprocess, thus I do not get why you need pypdftk (by the way, why not use this library???)